src/USDA.NET/Extensions/EnumExtensions.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Linq;
+using System.Reflection;
+
+namespace USDA.NET.Extensions
+{
+    public static class EnumExtensions
+    {
+        /// <summary>
+        /// Retrieves the DescriptionAttribute value from an enum value.
+        /// </summary>
+        /// <param name="value">The enum value</param>
+        /// <returns>The DescriptionAttribute value or if not defined, value.ToString()</returns>
+        public static string ToDescription(this Enum value)
+        {
+            return value.GetType()
+                .GetRuntimeFields()
+                .FirstOrDefault(x => x.Name == value.ToString())
+                ?.GetCustomAttribute<DescriptionAttribute>()
+                ?.Description ?? value.ToString();
+        }
+    }
+
+    //polyfil for System.ComponentModel.DescriptionAttribute to keep dependencies down
+    public sealed class DescriptionAttribute : Attribute
+    {
+        /// <summary>
+        /// <para>Specifies the default value for the <see cref='System.ComponentModel.DescriptionAttribute'/> , which is an
+        ///    empty string (""). This <see langword='static'/> field is read-only.</para>
+        /// </summary>
+        public static readonly DescriptionAttribute Default = new DescriptionAttribute();
+
+        public DescriptionAttribute() : this(string.Empty)
```

Comment:
What use case does this no-argument constructor cover? The below seems non-desirable:
```
public enum Value
{
  [Description] // provides no value
  One,

  [Description("Two")]
  Two
}
```

Reply:
All of your concerns over this class maybe warranted, however I copy & pasted this straight from the dot net core source code in order to keep System.ComponentModel from being a dependency.